R/shared_constants.R

```diff
@@ -228,6 +228,68 @@ strip_names <- function(x) {
   x
 }
 
+extract_glued_symbols <- function(expr, interpret_glue) {
```

Comment:
Add roxygen?
Maybe add tests so we can later expand more glue() like functions?

Reply:
> Maybe add tests so we can later expand more glue() like functions?

Not sure I follow, there are already a lot of tests in `object_usage_linter()`:

https://github.com/r-lib/lintr/blob/a59c6c06229c6e9c71d910fe31d379a554255bea/tests/testthat/test-object_usage_linter.R#L345-L538

Reply:
My thinking was that since this now got moved, we might more explicitly nail down the interface. Fine without tests, too. But roxygen should be there I think (with `@noRd`)